kdl_parser/CMakeLists.txt

```diff
@@ -30,6 +30,10 @@ target_link_libraries(${PROJECT_NAME}
   ${TinyXML_LIBRARIES} ${orocos_kdl_LIBRARIES} ${catkin_LIBRARIES}
 )
 
+if(WIN32)
+  target_compile_definitions(${PROJECT_NAME} PRIVATE "KDL_PARSER_BUILDING_DLL")
```

Comment:
GenerateExportHeader?

Reply:
Same basic idea as https://github.com/ros/urdf/issues/11, where William's feedback is interesting.  I can open an issue here as well.